lib/active_interactor/organizer/interactor_interface.rb

```diff
@@ -78,8 +83,31 @@ def execute_inplace_callback(target, callback)
         resolve_option(target, callbacks[callback])
       end
 
+      def execute_after_perform_callbacks(context)
+        return unless after_perform_callbacks.present?
+
+        interactor = interactor_class.new(context)
+        env = ActiveSupport::Callbacks::Filters::Environment.new(interactor, false, nil)
+        after_perform_callbacks.compile.invoke_after(env)
+        interactor.send(:context)
+      end
+
       private
 
+      def init_after_perform_callbacks
+        after_callbacks_deferred = @interactor_class.present? 
+                                && @interactor_class.after_callbacks_deferred_when_organized
```

Comment:
I believe the `&&` needs to be moved to the end of line 98, I'm unable to test atm whether this is an actual issue or the linter being picky but I am inclined to just go with what the linter asks of us.

```
SyntaxError:
  /home/runner/work/activeinteractor/activeinteractor/lib/active_interactor/organizer/interactor_interface.rb:99: syntax error, unexpected &&, expecting keyword_end
  ...                             && @interactor_class.after_call...
  ...                             ^~
```